rs/pocket_ic_server/src/main.rs

```diff
@@ -139,13 +164,27 @@ async fn start(runtime: Arc<Runtime>) {
         None
     };
 
+    let _guard = setup_tracing(args.log_levels);
+
+    // Set RUST_MIN_STACK if not yet set:
+    // the value of 8192000 is set according to `ic-os/components/ic/ic-replica.service`.
+    std::env::set_var("RUST_MIN_STACK", "8192000");
+
+    // Set the maximum number of open files:
+    // the limit of 16777216 is set according to `ic-os/components/ic/ic-replica.service`.
+    if let Err(e) = increase_nofile_limit(16777216) {
+        error!(
+            "Failed to increase the maximum number of open files: {:?}",
+            e
+        );
+    }
```

Comment:
The motivation for only performing the settings here is twofold:
- do not change the settings if the PocketIC binary exits early because another PocketIC binary is already running;
- use the tracing library for logging.

It is sufficient to only change the settings in the entry point of the PocketIC binary because they apply transitively to all child processes (i.e., also all the sandbox processes).